error.go

```diff
@@ -26,7 +26,7 @@ func (err redisError) Error() string {
 }
 
 func isNetworkError(err error) bool {
-	if _, ok := err.(*net.OpError); ok || err == io.EOF {
+	if _, ok := err.(net.Error); ok || err == io.EOF {
```

Comment:
Actually, not sure if this is even a concern, but would that include read/write timeouts? A tiny ReadTimeout could trigger every command to be retried although it actually succeeds.